web/pgadmin/tools/schema_diff/static/js/components/SchemaDiffButtonComponent.jsx

```diff
@@ -182,32 +191,32 @@ export function SchemaDiffButtonComponent({ sourceData, targetData, selectedRowI
         open={openMenuName == MENUS.COMPARE}
         onClose={onMenuClose}
         label={gettext('Compare')}
+        align="end"
       >
-        <PgMenuItem onClick={() => { selectCompareOption(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_OWNER); }}>
-          {selectedCompare.includes(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_OWNER) ? <CheckRoundedIcon /> : <span className={classes.emptyIcon}></span>}{gettext('Ignore owner')}
-        </PgMenuItem>
-        <PgMenuItem onClick={() => { selectCompareOption(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_WHITESPACE); }}>
-          {selectedCompare.includes(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_WHITESPACE) ? <CheckRoundedIcon /> : <span className={classes.emptyIcon}></span>}{gettext('Ignore whitespace')}
-        </PgMenuItem>
+        <PgMenuItem hasCheck checked={selectedCompare.includes(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_OWNER)}
+          onClick={() => { selectCompareOption(MENUS_COMPARE_CONSTANT.COMPARE_IGNORE_OWNER); }}>{gettext(IGNORE_OPTION.OWNER)}</PgMenuItem>
```

Comment:
gettext does not work on variables. Either use string or change IGNORE_OPTION.OWNER to use gettext.